app/Services/GitHub/Concerns/RendersIssueDetails.php

```
@@ -0,0 +1,161 @@
+    }
+    
+    /**
+     * Format labels with colors
+     */
+    protected function formatLabels(array $labels): array
+    {
+        $formatted = [];
+        
+        foreach ($labels as $label) {
+            $name = $label['name'];
+            $color = $label['color'] ?? '000000';
```

PHPMD (2.15.0)

Avoid unused local variables such as '$color'. (Unused Code Rules)

(UnusedLocalVariable)